### PR TITLE
fix(scripts): use MTP from the block where outpoint is created

### PR DESCRIPTION
**Pull request by m-kus**
Follow up #239

### CODE REVIEW

**Comment by m-kus** [comment]
@maciejka the script probably needs to be updated depending on how MTP field is named in your new dataset

**Comment by maciejka** [comment]
> @maciejka the script probably needs to be updated depending on how MTP field is named in your new dataset

Field is called 'median_timestamp'.

File: scripts/data/generate_data.py [hunk 1]
```diff
@@ -233,12 +195,12 @@ def format_outpoint(previous_output, median_time_past):
             "cached": False,
         },
         "block_height": int(previous_output["block_height"]),
-        "median_time_past": median_time_past,
+        "median_time_past": int[previous_output["median_time_past"]],
```
Comment (maciejka) [inline scripts/data/generate_data.py:198]: it should be `previous_output["median_timestamp"]`

Comment (m-kus) [reply scripts/data/generate_data.py:198]: It depends on whether MTP comes from RPC or bigquery file
Ok

**Review by maciejka — APPROVED** [review]
LGTM

**Pull request merged**